mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -170,21 +170,24 @@ final class OneVsRestModel private[ml] (
       newDataset.persist(StorageLevel.MEMORY_AND_DISK)
     }
 
+    // temporary column to store intermediate raw prediction
+    val tmpRawPredictionColName = "mbc$tmpraw" + UUID.randomUUID().toString
```

Comment:
in other ml cases we are slightly more descriptive with the prefix text

https://github.com/apache/spark/blob/576c43fb4226e4efa12189b41c3bc862019862c6/mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala#L1050